src/components/FilterPanel/index.js

```diff
@@ -50,6 +50,31 @@ const useStyles = makeStyles((theme) => ({
     },
 }));
 
+function ClearSelections({ classes, inProgressFilters, setInProgressFilters }) {
+    return (
+        <Button
+            variant="outlined"
+            color="default"
+            className={classes.mobileButton}
+            onClick={(e) => {
+                e.preventDefault();
+                setInProgressFilters({
+                    ...inProgressFilters,
+                    filterByAvailable: true,
+                    filterByZipCode: {
+                        ...inProgressFilters.filterByZipCode,
+                        zipCode: "",
```

Comment:
Should we unset the cookie too? Might be good to move the cookie logic to whenever someone presses either button

Reply:
As it is right now, this button isn't any different from clicking checboxes and clearing numbers.  It doesn't press the "Update List" button.